tools/test_runner/show_reports.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# This script runs HTTP server for small test reports
+# Usage:
+# - ./tools/test_runner/show_reports.sh small
+# - ./tools/test_runner/show_reports.sh big
+set -e
+
+# Go to the repo directory
+cd "$( dirname "${BASH_SOURCE[0]}" )"/../..
+
+# Usage: run_http_server_for PORT DIRECTORY
+function run_http_server_for
+{
+    local PORT="$1"
+    local DIRECTORY="$2"
+    mkdir -p "$DIRECTORY"
+    local ABS_DIRECTORY=$(./tools/abs_dirpath.sh "$DIRECTORY")
+    cd "$DIRECTORY"
+
+    echo "Running an HTTP server for $ABS_DIRECTORY"
+    python -m SimpleHTTPServer "$PORT" >> http_server.log 2>> http_server.log &
+    echo "Open http://localhost:$PORT in your browser"
```

Comment:
1. There is no need to for the python's HTTP server, we can open the `${ABS_DIRECTORY}/index.html` or `${ABS_DIRECTORY}/all_runs.html`.
2. Instead of only printing `Open sth in your browser` we can simply call
    ```bash
    open  ${ABS_DIRECTORY}/index.html
    ```
   and this will open the report in the default browser. I would still leave the URL printing so that the user sees in the console what was opened.

Reply:
not really, chrome does not load xslt stylesheets without a browser.

"open" would not work on a remote shell or with a separate user for code.

`open` is also platform-specific.

Yeah, I can try to call `open` and `xdg-open`, but it would not always work.

Reply:
Ok, let's leave it as is. Thanks for the explanation.